boeing_voucher/admin.py

```diff
@@ -0,0 +1,3 @@
+"""
```

Comment:
Maybe change the name of this app from `boeing_voucher` to just `voucher` and remove the company from docstrings & class names, just in case they're sensitive about that?

Reply:
@Ferdi Can you weigh in on this one?

Reply:
@sesterki it would be great if you can

Reply:
Done